config/jobs/kubernetes-sigs/kueue/kueue-presubmits.yaml

```diff
@@ -60,7 +60,7 @@ presubmits:
       preset-dind-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220903-da891f21aa-master
```

Comment:
probably need to set the environment variable too?

Although that's when the presubmit might fail for older releases :(

Reply:
> probably need to set the environment variable too?

Does this mean ` DOCKER_BUILDX_CMD`?

Reply:
No, it should be `IMAGE_BUILD_CMD`, the one from the `image-build` directive.


Maybe we can add two presubmits, one for old branches (release-0.1 and release-0.2) and one for the rest. This should be possible:

https://github.com/kubernetes/test-infra/blob/1d9942e1a77c183f41c809dc9a767ff220d5fe48/prow/jobs.md#how-to-configure-new-jobs

Reply:
Thanks for clarifying.
That makes sense. I agree with you.

Reply:
Done.